### PR TITLE
added server_model_directory argument in server.py

### PR DESCRIPTION
**Pull request by suryanshagnihotri**
Added server_model_arg in rsa_nlu/server.py.
It was giving error when using `python -m rasa_nlu.server -c config_spacy.json --server_model_dirs=./model_YYYYMMDD-HHMMSS` since no --server_model_dirs arg is present in server.py.

### CODE REVIEW

File: rasa_nlu/server.py [hunk 1]
```diff
@@ -44,6 +44,8 @@ def create_argparser():
     parser.add_argument('--pipeline',
                         help="The pipeline to use. Either a pipeline template "
                              "name or a list of components separated by comma")
+    parser.add_argument('--server_model_dirs',
+                        help="address of server_model_directory")
```
Comment (tmbo) [inline rasa_nlu/server.py]: Please use the description from https://rasahq.github.io/rasa_nlu/0.9.2/config.html#server-model-dirs (or something more descriptive than the current sentence :D)

Comment (suryanshagnihotri) [reply rasa_nlu/server.py]: Done :)

**Comment by PHLF** [comment]
I removed it because the default server behavior since #449 is to look for all folders under the `path` directory and expose them as endpoints thus making this setting no longer relevant.

**Comment by tmbo** [comment]
@PHLF right - thanks, I didn't remember that :+1:. @suryanshagnihotri we don't need this configuration variable in the latest version then, the `-path` parameter that already exists on the server script should solve your problem.

**Pull request closed without merging**